sdk/cosmos/azure-cosmos/test/query_tests.py

```diff
@@ -214,6 +218,177 @@ def _MockExecuteFunction(self, function, *args, **kwargs):
         self.count += 1
         return self.OriginalExecuteFunction(function, *args, **kwargs)
 
+    def test_get_query_plan_through_gateway(self):
+        created_collection = self.config.create_multi_partition_collection_with_custom_pk_if_not_exist(self.client)
+        self._validate_query_plan("Select top 10 value count(c.id) from c", created_collection.container_link, 10, [], ['Count'], True, None, None, "None")
```

Comment:
Please annotate these literals, so we can know what is what.

Reply:
cool, will do in next iteration